core/eval.go

```diff
@@ -856,6 +857,97 @@ func evalQWATCH(args []string, c *Client) []byte {
 	return RESP_OK
 }
 
+func evalBITCOUNT(args []string) []byte {
+	var err error
+
+	// if more than 4 arguments are provided, return error
+	if len(args) > 4 {
+		return Encode(errors.New("ERR invalid number of arguments for `evalBITCOUNT` command"), false)
+	}
+
+	// fetching value of the key
+	var key string = args[0]
+	var obj = Get(key)
+	if obj == nil {
+		return Encode(errors.New("ERR key does not exist for `evalBITCOUNT` command"), false)
+	}
+	var valueInterface = obj.Value
+	if err := assertType(obj.TypeEncoding, OBJ_TYPE_STRING); err != nil {
+		return Encode("ERR string value do not exists on key", false)
+	}
+	value := []byte(valueInterface.(string))
```

Comment:
Give this command is meant for ByteArray Data type, let's check for that instead of string. Some utility functions are already written there.